lib/util.js

```diff
@@ -433,7 +436,7 @@ function formatValue(ctx, value, recurseTimes, ln) {
     return ctx.stylize('null', 'null');
   }
 
-  if (ctx.showProxy) {
+  if (ctx.showProxy && !nativeModuleProxies.has(value)) {
```

Comment:
I don't think we should make a distinction between "our proxies" versus "their proxies".

We should either disable proxy showing by default, or just show the proxy.

Reply:
@TimothyGu It's cosmetic and can totally be tackled in a follow-up PR. The idea from earlier in the thread was that folks thought repl inspection would be less-good displaying the `Proxy` prefix when inspecting the builtin module export.

Reply:
I'd like to argue in the other way: because it is cosmetic, this change can be done in a follow-up PR. Changes should be atomic.

Reply:
> I'd like to argue in the other way: because it is cosmetic, this change can be done in a follow-up PR.

We're on the same page. I was saying that the masking of builtin exports could be done in a follow-up PR instead of this one 😁

Reply:
Oops 😛